crates/swc/tests/tsc-references/exportsAndImports1-amd_es2015.1.normal.js

```diff
@@ -31,5 +31,5 @@ var M;
 var a = M.x;
 export { v, f, C, E, D, a };
 // @filename: t2.ts
-export { v, f, C, E, D, a } from "./t1";
+export { v, f, C, I, E, D, M, N, T, a } from "./t1";
```

Comment:
This was actually bugged me when I work on my pr: I expected namespace exports are getting stripped (as same as current fixture) - do you think chanhed behavior is expected?

Reply:
https://github.com/swc-project/swc/blob/fc9622f779c60d9006ed54710b13b127674c6574/crates/swc_ecma_parser/tests/typescript/tsc/exportsAndImports1-amd/input.ts

Actually, it's difference of original test runner of tsc (supports `@filename`) vs test runner of swc (ignores `@filename`).
In this case, preserving exports is fine. This will not negatively effect real-world code.

Reply:
Thanks for the clarification. I struggled several hr to preserve this one and I should've asked before instead.